docs/storage.md

```diff
@@ -1,43 +1,43 @@
 # Storage
 
-Parca has two distinct storage components: metastore and samplestore. The metastore stores metadata such as function names, line numbers, file names, and so on. The samplestore stores the profiling data.
+Parca has two distinct storage components: meta store and sample store. The meta store stores metadata such as function names, line numbers, file names, and so on. The sample store stores the profiling data.
 
-## Metastore
+## Meta Store
```

Comment:
FYI we might have used `metastore` as a single word everywhere in the docs and other repositories.

Reply:
I was wondering if there is a convention we follow.
Having the metastore next to the sample store seems wrong and thus I went with meta store. hmm.

Reply:
I don't mind which one to use, as long as we're consistent :)